po/es.po

```diff
@@ -72,25 +72,25 @@ msgstr "etiqueta Nombre de Escritorio vacía"
 
 #: src/dock.c:150
 msgid "only one Dock allowed"
-msgstr "solo se permite un Dock"
+msgstr "solo se permite un Muelle"
```

Comment:
The word `Dock` refers to the XML tag, which is why its title case.  I suppose that probably makes less sense in other languages, but I wonder if it should remain untranslated?